README.md

```diff
@@ -49,9 +49,9 @@ pip install edvart
 
 ## Usage
 
+
 See the notebook
-[api-example.ipynb](https://nbviewer.org/github/datamole-ai/edvart/blob/main/api-example.ipynb)
-for usage examples.
+[examples/report-example.ipynb](https://nbviewer.org/github/datamole-ai/edvart/blob/main/examples/report-example.ipynb) for an example report or [examples/time-series-report-time-series-report-example.ipynb](https://nbviewer.org/github/datamole-ai/edvart/blob/main/examples/time-series-report-example.ipynb)
```

Comment:
missing end of sentence? `or ... for an example time series report`?

Reply:
Yup, fixed :facepalm: